packagebuild/daisy_startupscript_deb.sh

```diff
@@ -42,6 +44,14 @@ try_command apt-get install -y --no-install-{suggests,recommends} git-core \
 
 git_checkout "$REPO_OWNER" "$REPO_NAME" "$GIT_REF"
 
+if [[ ! -z "$EXTRA_REPO" ]]; then
```

Comment:
Do we need to install protoc?

Reply:
Adding as dependency for `guest-agent` control file.

Reply:
On second though, installing it as part of this script. `debian-11` doesn't have `protoc-gen-go-grpc` in its `apt` repositories, so this is required for that anyways.

Reply:
actually, we just introduced common [install_protoc](https://github.com/GoogleCloudPlatform/guest-test-infra/blob/eb8ab86e71a42dd023c296cc10f725eafd477f64/packagebuild/common.sh#L32) for that. You can use it here so it will be consistent across packaging